src/DashboardUI/src/pages/application/review/ApplicationReviewPage.tsx

```diff
@@ -111,3 +164,51 @@ function ReviewerButtonRow(props: ReviewerButtonRowProps) {
     </div>
   );
 }
+
+interface SaveChangesModalProps {
+  show: boolean;
+  onCancel: () => void;
+  onConfirm: (documentedChanges: string) => void;
+}
+
+function SaveChangesModal(props: SaveChangesModalProps) {
+  const [documentedChanges, setDocumentedChanges] = useState('');
+
+  return (
+    <Modal show={props.show} centered>
+      <Modal.Header closeButton onClick={props.onCancel}>
+        <Modal.Title>Document your Changes</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <p>
+          To save any modifications to this application, please{' '}
+          <strong>record what content you edited or added.</strong> Once submitted, your changes can be seen by you and
+          other reviewing users at the bottom of this application.
+        </p>
+
+        <Form>
+          <Form.Group controlId="documentedChanges">
+            <Form.Label>Changes</Form.Label>
+            <Form.Control
+              as="textarea"
+              type="text"
+              maxLength={4000}
```

Comment:
Remove label and use attribute to match mockups

```suggestion
            <Form.Control
              as="textarea"
              aria-label="Changes"  // Use whatever the correct aria label attr is here
              type="text"
              maxLength={4000}
```

Reply:
thanks, updating